### DISCUSSION TITLE
Anchorsearch

### DISCUSSION THREAD

**Discussion by og900aero** [gokcehan/lf#1710] [Q&A] [answered]
There is a setting for find command for find starts matching patterns from the beginning of file names, otherwise, it can match at an arbitrary position. This is the anchorfind. Is there such a setting for the search command? Because if it isn't there, it would be useful if it had a name like this: anchorsearch

**Comment by CatsDeservePets** [comment, marked as answer]
Hello @og900aero!

This is already possible by using `set globsearch`. Now you can just search for something like `hello*` to only match file names starting with `hello`.
